### DISCUSSION TITLE
支付宝链接无法使用

### DISCUSSION THREAD

**Discussion by businiaozhimen** [getlantern/lantern-forum-cn#597] [Q&A]
支付宝链接无法使用 账户 还有15天过期
<img width="796" alt="捕获" src="https://github.com/getlantern/lantern-forum-cn/assets/173088681/e1918eab-64f7-4cf6-b1c6-141a9fb8bcc2">

**Comment by MaxMKMax** [comment]
@businiaozhimen 嗯嗯. 第三方的支付系统有问题. 我们在处理着了. 请您过两天再试看付款. 谢谢您支持蓝灯.

**Reply by businiaozhimen** [reply]
什么时候处理好 还有12天就过期了

**Reply by businiaozhimen** [reply]
@Mike777Khoo 什么时候处理好 可以使用支付宝

**Reply by businiaozhimen** [reply]
@Mike777Khoo 什么时候可以处理好 还有10天过期

**Reply by MaxMKMax** [reply]
@businiaozhimen 
您好. 好像已经处理好了. 请您再试试看多几次. 谢谢您支持蓝灯!

**Reply by businiaozhimen** [reply]
@Mike777Khoo 还是同样报错

**Reply by businiaozhimen** [reply]
@Mike777Khoo 什么时候支付宝可以付款 还有5天过期

**Comment by MaxMKMax** [comment]
@businiaozhimen 对的. 第三方支付系统有些问题导致支付宝无法使用. 我们在解决这问题了. 谢谢您的耐心等待.

**Reply by businiaozhimen** [reply]
@Mike777Khoo 什么时候支付宝可以付款 还有3天过期

**Reply by businiaozhimen** [reply]
@Mike777Khoo 什么时候支付宝可以付款 还有1天过期

**Comment by businiaozhimen** [comment]
HI Mike K

 

现在付款链接都无法打开了

 



 

 

发件人: ***@***.*** ***@***.***> 代表 Mike K
发送时间: 2024年6月22日 3:04
收件人: getlantern/lantern-forum-cn ***@***.***>
抄送: businiaozhimen ***@***.***>; Mention ***@***.***>
主题: Re: [getlantern/lantern-forum-cn] 支付宝链接无法使用 (Discussion #597)

 

@businiaozhimen <https://github.com/businiaozhimen> 
您好. 好像已经处理好了. 请您再试试看多几次.
